src/pages/Account/Login.scss

```diff
@@ -29,6 +29,15 @@
   }
 
   &__alternative-logins {
+    outline: none !important;
```

Comment:
Could we avoid `!important`?

Reply:
Why are we taking away outline at all? As I've mentioned before, **outline** is a built-in accessibility feature, and we should not be removing it, even if it looks better.

This should not be a blocking comment, we need this merged asap, but please consider removing this line entirely @dmitrii-zolotukhin 

https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

Reply:
@yurii-lubynets @whoaitsaimz fixed in 5f6ea55285f3f28e51ae012db3290ac4694d5c03